package.json

```diff
@@ -1,14 +1,19 @@
 {
   "name": "recipy-book",
   "version": "1.0.0",
-  "babel": {},
+  "babel": {
```

Comment:
this is babel config for ther server. Config for the client is specified in common webpack config file right in the babel-loader section